src/ripple/app/main/Application.cpp

```diff
@@ -1027,12 +1034,15 @@ void ApplicationImp::setup()
                 startUp == Config::LOAD_FILE ||
                 startUp == Config::REPLAY)
     {
-        JLOG(m_journal.info()) << "Loading specified Ledger";
+        JLOG(m_journal.info()) <<
+            "Loading specified Ledger";
```

Comment:
I might be inclined to format this as `"Loading ledger " << config_->START_LEDGER`.